themes/dark/templates/editShow.mako

```diff
@@ -216,4 +220,130 @@
         </form>
     </div>
 </div>
+
+</%block>
+<%block name="scripts">
+    <script type="text/javascript" src="js/quality-chooser.js?${sbPID}"></script>
+    <script type="text/javascript" src="js/edit-show.js"></script>
+<script src="js/lib/vue.js"></script>
+<%include file="/vue-components/selectListUi.mako"/>
+<%include file="/vue-components/anidbReleaseGroupUi.mako"/>
+<script>
+var startVue = function() {
+    const app = new Vue({
+        el: '#config-content',
+        data() {
+            // Python conversions
+            // JS only
+            const exceptions = [];
+            return {
+                seriesSlug: $('#series-slug').attr('value'),
+                seriesId: $('#series-id').attr('value'),
+                indexerName: $('#indexer-name').attr('value'),
+                config: MEDUSA.config,
+                exceptions: exceptions,
+                seriesObj: {
```

Comment:
Please switch this to `series`. No need to describe the type in the name.

Reply:
done